src/node_options.h

```diff
@@ -44,6 +44,16 @@ class HostPort {
   uint16_t port_;
 };
 
+template <typename ValueType>
+class DetailedOption {
```

Comment:
Are there instances where we would need `DetailedOption` with a value that is not a string?

The template adds a little bit of noise and if it is entirely unnecessary, you could slightly simplify the code by just removing it.

(Note that it is always possible to generalize later if needed.)

Reply:
I've removed the templating